star-war-character-app/src/pages/Species/SpeciesDetails/SpeciesDetails.tsx

```diff
@@ -0,0 +1,100 @@
+import { useSelector } from "react-redux";
+import { IRootState, useAppDispatch } from "../../../redux/store";
+import { Fragment, useEffect } from "react";
+import { getIndividualSpeciesActions } from "../../../redux/SpeciesSlice/SpeciesAsyncThunk";
+import { Link, useParams } from "react-router-dom";
+import { speciesAction } from "../../../redux/SpeciesSlice/SpeciesSlice";
+import { Strings } from "../../../resource/Strings";
+import moment from "moment";
+import { Loader } from "../../../Loader/Loader";
+import "./SpeciesDetails.css"
+
+const SpeciesDetails = () => {
+  const { specificSpecies } = useSelector(
+    (state: IRootState) => state.speciesStateData
+  );
+  const dispatch = useAppDispatch();
+  const isLoading = useSelector(
+    (state: IRootState) => state.speciesStateData.isLoading
+  );
+  const { speciesId } = useParams();
+  useEffect(() => {
```

Comment:
Change the file name as per discussed.

Reply:
I have resolved this issue in this PR https://github.com/Harsh-Upadhyay-AT/Star-War-Character-App/pull/2 Please check it.